react-components/src/hooks/useSearchMappedEquipmentFDM.tsx

```diff
@@ -64,13 +65,22 @@ export const useSearchMappedEquipmentFDM = (
           result.items.mapped_nodes.concat(result.items.mapped_nodes_2) as NodeItem[]
         );
 
-        return transformedResults;
+        const combinedWithOtherViews = viewsToSearch.map((view) => ({
+          view,
+          instances:
+            transformedResults.find(
+              (result) =>
+                result.view.externalId === view.externalId && result.view.space === view.space
+            )?.instances ?? []
+        }));
```

Comment:
Just to understand - this makes sure that all views are associated with some results (if any), even if there are duplicate views in the `viewsToSearch` list?

Reply:
It just ensures that it returns "3d searchable" views even if there are no results for them